keyvi/src/cpp/dictionary/fsa/generator.h

```diff
@@ -203,7 +200,7 @@ final {
       ConsumeStack(commonPrefixLength);
 
       // put everything that is not common between the two strings (the suffix) into the stack
-      FeedStack(commonPrefixLength, input_key.size(), key);
+      FeedStack(commonPrefixLength, input_key.size(), input_key.c_str());
```

Comment:
What about changing signature of FeedStack to have param `const std::string& input_key`